<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayStatusDialog</class>
 <widget class="QDialog" name="PayStatusDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подтверждение оплаты</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="6" column="0">
    <widget class="QLabel" name="lblNote">
     <property name="text">
      <string>Примечание</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="edtNumber">
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblPayRefuseType">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Причина отказа</string>
     </property>
     <property name="buddy">
      <cstring>cmbRefuseType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblNumber">
     <property name="text">
      <string>Документ</string>
     </property>
     <property name="buddy">
      <cstring>edtNumber</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbRefuseType">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="rbnAccepted">
     <property name="text">
      <string>Оплачено</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QRadioButton" name="rbnRefused">
     <property name="text">
      <string>Отказано</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>81</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDate">
     <property name="text">
      <string>Дата</string>
     </property>
     <property name="buddy">
      <cstring>edtDate</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QRadioButton" name="rbnFactPayed">
     <property name="text">
      <string>Частично оплачено на сумму</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="edtNote"/>
   </item>
   <item row="7" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="edtFactPayed">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>99999999.989999994635582</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtDate</tabstop>
  <tabstop>edtNumber</tabstop>
  <tabstop>rbnAccepted</tabstop>
  <tabstop>rbnRefused</tabstop>
  <tabstop>cmbRefuseType</tabstop>
  <tabstop>edtNote</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PayStatusDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PayStatusDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbnRefused</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbRefuseType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbnRefused</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblPayRefuseType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbnFactPayed</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtFactPayed</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
